<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>358</width>
    <height>225</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>User Import</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="label">
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>30</height>
       </size>
      </property>
      <property name="text">
       <string>Upload Users JSON</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="upload_button">
      <property name="text">
       <string>Upload</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="file_label">
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>30</height>
       </size>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item alignment="Qt::AlignmentFlag::AlignRight">
     <widget class="QToolButton" name="next_button">
      <property name="text">
       <string>Next</string>
      </property>
     </widget>
    </item>
    <item alignment="Qt::AlignmentFlag::AlignRight">
     <widget class="QToolButton" name="skip_button">
      <property name="text">
       <string>Skip</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>358</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
